corehq/apps/app_manager/id_strings.py

```diff
@@ -90,6 +90,15 @@ def detail_column_header_locale(module, detail_type, column):
     )
 
 
+@pattern('m%d.%s.%s.display')
+def sort_only_column_header_locale(module, detail_type, sort_element):
+    return u"m{module.id}.{detail_type}.{field}.display".format(
+        detail_type=detail_type,
+        module=module,
+        d=sort_element,
```

Comment:
`d` is for module.id. the '0' in `m0.case_short.case_name_2.header`. I just mirrored the convention used for `detail_column_header_locale` as much as possible.